internal/schemamd/hidden.go

```diff
@@ -0,0 +1,10 @@
+package schemamd
+
+// hiddenFields is the list of fields we don't want to appear in the documentation
+// because users should not have to interact with them.
+var hiddenFields = []string{
+	"id",
+	"kind",
+	"metadata.namespace",
+	"metadata.revision",
+}
```

Comment:
This should be a configuration when running the generator

Reply:
Yes, but I'll have to rewrite the entire library to propagate options 😢

Reply:
Done